### PR TITLE
processor: prevents individual table pipelines to block the entire processor

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve? 
- In the current implementation of the new processor, if the pipeline for any table is blocked, the entire processor will be blocked.

### What is changed and how it works?
- Skip sending messages to the blocked pipeline. Note that this only means stop sending control messages to the table pipeline, and any back-pressure handling mechanism for the data critical path remains unchanged.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test

Start 10 tpcc changefeeds, and wait for incremental scans to finish.
`curl "http://xxx:8300/debug/fail/github.com/pingcap/ticdc/pkg/pipeline/PipelineSendToFirstNodeTryAgain" -X PUT -d '40%return(true)'`
Wait for 1 minute and review log to see that the failpoint has taken effect.
`curl "http://xxx:8300/debug/fail/github.com/pingcap/ticdc/pkg/pipeline/PipelineSendToFirstNodeTryAgain" -X PUT -d 'return(true)'`
Wait for 1 minute to observe that checkpoint-ts has stopped updating.
`curl "http://xxx:8300/debug/fail/github.com/pingcap/ticdc/pkg/pipeline/PipelineSendToFirstNodeTryAgain" -X DELETE`
Wait until checkpoint-ts resumes updating.

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

**Review by leoppro — APPROVED** [review]

File: cdc/processor/processor.go [hunk 1]
```diff
@@ -365,7 +365,11 @@ func (p *processor) handleTableOperation(ctx cdcContext.Context) error {
 				if opt.BoundaryTs < globalCheckpointTs {
 					log.Warn("the BoundaryTs of remove table operation is smaller than global checkpoint ts", zap.Uint64("globalCheckpointTs", globalCheckpointTs), zap.Any("operation", opt))
 				}
-				table.AsyncStop(opt.BoundaryTs)
+				if !table.AsyncStop(opt.BoundaryTs) {
```
Comment (amyangfei) [inline cdc/processor/processor.go:368]: If table `AsyncStop` returns false here, will it be called later?

Comment (liuzix) [reply cdc/processor/processor.go:368]: Yes. `EtcdWorker` will tick regularly and the processor will attempt to handle unfinished operations each tick.

**Review by amyangfei — APPROVED** [review]

**Pull request merged**